src/Fixer/ArrayNotation/TrimArraySpacesFixer.php

```diff
@@ -71,12 +71,14 @@ private static function fixArray(Tokens $tokens, $index)
             $endIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_ARRAY_SQUARE_BRACE, $startIndex);
         }
 
-        $nextToken = $tokens[$startIndex + 1];
+        $nextIndex = $startIndex + 1;
+        $nextToken = $tokens[$nextIndex];
         $nextNonWhitespaceIndex = $tokens->getNextNonWhitespace($startIndex);
         $nextNonWhitespaceToken = $tokens[$nextNonWhitespaceIndex];
         $tokenAfterNextNonWhitespaceToken = $tokens[$nextNonWhitespaceIndex + 1];
 
-        $prevToken = $tokens[$endIndex - 1];
+        $prevIndex = $endIndex - 1;
+        $prevToken = $tokens[$prevIndex];
```

Comment:
`$prevToken` variable is now used only once, so you can pass its value directly instead.